### PR TITLE
Update WG-prioritization page

### PR DESCRIPTION
**Pull request by apiraino**
@spastorino I've tried updating this page, I think there is some content outdated.

Can you have a look if my changes make sense?

Thanks :-)

### CODE REVIEW

File: content/working-groups/prioritization/_index.md
Comment (apiraino) [inline content/working-groups/prioritization/_index.md:44]: I think this whole section is outdated, right? I've never seen these labels.

File: content/working-groups/prioritization/_index.md
Comment (apiraino) [inline content/working-groups/prioritization/_index.md:42]: if I am not wrong, meetings time are different from the calendar now

**Review by apiraino — COMMENTED** [review]
see my comments around

File: content/working-groups/prioritization/_index.md
Comment (spastorino) [inline content/working-groups/prioritization/_index.md:35]:
```suggestion
`P-critical`. But if unclear, use the `I-prioritize` label to
```

File: content/working-groups/prioritization/_index.md
Comment (spastorino) [reply content/working-groups/prioritization/_index.md:44]: I have no idea, maybe this was an idea that we wanted to happen at some point?. cc @pnkfelix

File: content/working-groups/prioritization/_index.md
Comment (spastorino) [reply content/working-groups/prioritization/_index.md:42]: This is wrong, the text was talking about prioritization wg meetings which do not happen anymore. The text can just say, `If you are interested in getting involved in this working group, come and say hi in the Zulip stream.` with a link to it.

File: content/working-groups/prioritization/_index.md
Comment (spastorino) [inline content/working-groups/prioritization/_index.md:50]: This is wrong too, meetings do not happen anymore. You're confusing prioritization wg meetings with the compiler triage meetings which are the ones that we build an agenda for.

**Review by spastorino — CHANGES_REQUESTED** [review]

**Comment by spastorino** [comment]
@apiraino left some comments but in general agree that this page is very outdated. We may also want to make even deeper changes to this :).

**Comment by apiraino** [comment]
> @apiraino left some comments but in general agree that this page is very outdated. We may also want to make even deeper changes to this :).

thanks for the review. I'll apply your suggestions and either merge or - let me know if you prefer - wait for a further review with you. It could be nice for newcomers to have the updated overview.

(I'd usually go with small incremental changes, but either way is fine)

**Comment by spastorino** [comment]
@apiraino I think you can apply my suggestions and after this is merged we can give it another pass.

File: content/working-groups/prioritization/_index.md
Comment (apiraino) [reply content/working-groups/prioritization/_index.md:44]: if not yet implemented, I'd probably remove it until it is (perhaps less confusing).

**Comment by apiraino** [comment]
> @apiraino I think you can apply my suggestions and after this is merged we can give it another pass.

I think it's done. I have also added some git repositories that are related to the workgroup.

Can't test the rendered version because some weird `hugo` compile issue I didn't not yet looked into.

**Review by spastorino — APPROVED** [review]

**Pull request merged**
